tests/test_canonicalise.py

```diff
@@ -192,6 +192,13 @@ def test_canonicalises_to_expected(schema, expected):
         ([{"type": "null"}, {"enum": [None, True]}], {"const": None}),
         ([{"type": "null"}, {"type": ["null", "boolean"]}], {"const": None}),
         ([{"type": "integer"}, {"maximum": 20}], {"type": "integer", "maximum": 20}),
+        (
+            [
+                {"type": "integer", "mutlipleOf": 3},
+                {"type": "integer", "multipleOf": 6},
+            ],
+            {"type": "integer", "multipleOf": 6},
```

Comment:
```suggestion
            [
                {"type": "integer", "multipleOf": 4},
                {"type": "integer", "multipleOf": 6},
            ],
            {"type": "integer", "multipleOf": 12},
```

- Using `4` instead of `3` checks that we're not just discarding multipliers which are a factor of another multiplier
- The confusing error you were getting is thanks to a typo!  Transposed "muTLiplier" in the first keyword.